lib/service/src/test/java/io/atlasmap/service/MappingServiceTest.java

```diff
@@ -146,4 +148,11 @@ public void testProcessMappingCustomAction() throws Exception {
         assertEquals("param foo", pmr.getMapping().getOutputField().get(0).getValue());
     }
 
+    @Test
```

Comment:
Can you add a few assertion to verify the field is removed? you can use `MappingService#getMappingRequest()` to get the current mapping JSON.

Reply:
I attempted to address the MappingServiceTest issue.  The real issue is that the processMappingRequest function doesn't actually process the URI.  I think the correct solution here is to have a generic method for executing the REST URI Info string.  I think this will require a new issue.

Reply:
I didn't notice earlier, but shouldn't it be `mappingService.removeMappedFieldRequest()` for this test? `mappingService.processMappingRequest()` is to execute the Mapping Preview IIRC, and basically URIInfo shouldn't be such important when you invoke Java method directly.

Reply:
Looking at the `MappingService#removeFieldMappingRequest()`
https://github.com/atlasmap/atlasmap/pull/3999/files#diff-5f102bdf1096d7b5c32dcda1c4506365f5edb58bb73ee7fd9059b1e91703751eR241
It doesn't even have URIInfo as a parameter, so you can't pass it in I think.

Reply:
Correct - so I don't have a mapping that's been digested.  I was looking for a generic way of calling into these functions with a predefined JSON mapping.  wdyt?

Reply:
cf. `MappingService#processMappingRequest()` is for processing Mapping Preview
https://github.com/atlasmap/atlasmap/pull/3999/files#diff-5f102bdf1096d7b5c32dcda1c4506365f5edb58bb73ee7fd9059b1e91703751eL397

Reply:
yes - I get that - the issue I'm having is that `MappingService#removeFieldMappingRequest()` will fail because the `AtlasMappinmgHandler` is never set up.  So given a test like this:
<pre>
    public void testRemoveFieldMappingRequest() throws Exception {
    	Response res = mappingService.removeFieldMappingRequest(0, "mapping.xxxxxx", DataSourceType.TARGET, 1);
        assertEquals(200, res.getStatus());
    }
</pre>
Fails - I need to register the mapping first...

Reply:
You can use `AtlasService#importADMArchiveRequest()` just like this
https://github.com/atlasmap/atlasmap/blob/main/lib/service/src/test/java/io/atlasmap/service/DocumentServiceTest.java#L95
Or you can use Mockito to mock up ADMArchiveHandler/AtlasMappingHandler and let them handle a dummy Mapping Definition.

Reply:
Just `MappingService#updateMappingRequest()` might also work
https://github.com/atlasmap/atlasmap/blob/main/lib/service/src/main/java/io/atlasmap/service/MappingService.java#L312